storage/jellyfish-merkle/src/node_type/mod.rs

```diff
@@ -388,11 +386,15 @@ impl InternalNode {
                 .unwrap()
                 .hash
         } else {
-            let left_child = self.merkle_hash(start, width / 2, (existence_bitmap, leaf_bitmap));
+            let left_child = self.merkle_hash(
+                start,
+                width / 2,
+                (range_existence_bitmap, range_leaf_bitmap),
+            );
             let right_child = self.merkle_hash(
                 start + width / 2,
                 width / 2,
-                (existence_bitmap, leaf_bitmap),
+                (range_existence_bitmap, range_leaf_bitmap),
```

Comment:
why?

Reply:
recursion func 
so this is more reasonable

Reply:
I don't get it. It seems a noop to me. Why do you think it is more reasonable? They are both passed by value and the mask will only keep the bits in the "window" for each recursive call.

Reply:
for example  
start 0 width 8
0x0110_1001_1111_0101  0x0110_1000_1111_0001   
origin
0x0110_1001_1111_0101  0x0110_1000_1111_0001
0x0110_1001_1111_0101  0x0110_1000_1111_0001
update:
0x0110_0000_0000_0000  0x0110_0000_0000_0000
0x0000_1001_0000_0000  0x0000_1000_0000_0000

Reply:
They both produce the same result on line 359.

Reply:
origin was no problem
update change, it is double insurance, which is safer